### PR TITLE
Fix setup_all and e2e test

### PR DESCRIPTION
**Pull request by jonchenn**
- Fixed typos in setup_all.sh
- Removed pytest in setup_all.sh
- Updated run_e2e_test.sh with pytest

### CODE REVIEW

File: e2e/run_e2e_test.sh [hunk 1]
```diff
@@ -27,6 +27,19 @@
 # To calculate time elapsed.
 SECONDS=0
 
+# Hardcoded the project ID for all local development.
+declare -a EnvVars=(
+  "ORGANIZATION_ID"7
```
Comment (lavjain) [inline e2e/run_e2e_test.sh]: There is an extra 7 (typo) here

Comment (jonchenn) [reply e2e/run_e2e_test.sh]: Thanks! Fixed.

File: setup/setup_all.sh [hunk 2]
```diff
@@ -39,7 +39,10 @@ setup_gcloud() {
 
 # Updating GCP Organizational policies
 update_gcp_org_policies() {
-  export ORGANIZATION_ID="$(gcloud organizations list --format='value(name)' | head -n 1)"
+  if [[ "$ORGANIZATION_ID" ==  "" ]]; then
+    export ORGANIZATION_ID="$(gcloud organizations list --format='value(name)' | head -n 1)"
```
Comment (lavjain) [inline setup/setup_all.sh:43]: The organization id may not be the first entry

File: setup/setup_all.sh [hunk 3]
```diff
@@ -55,7 +58,10 @@ grant_storage_iam() {
 
 # Link billing account to the current project.
 link_billing_account() {
-  export BILLING_ACCOUNT=$(gcloud beta billing accounts list --format "value(name)" | head -n 1)
+  if [[ "$BILLING_ACCOUNT" ==  "" ]]; then
+    export BILLING_ACCOUNT=$(gcloud beta billing accounts list --format "value(name)" | head -n 1)
```
Comment (lavjain) [inline setup/setup_all.sh:62]: We should ask user for billing account as there may be multiple and first one may not be correct

**Review by lavjain — APPROVED** [review]

File: setup/setup_all.sh (hunk 3 above)
Comment (jonchenn) [reply setup/setup_all.sh:62]: Yes, it's possible. We'll add more details in the README in later PR.

File: setup/setup_all.sh (hunk 2 above)
Comment (jonchenn) [reply setup/setup_all.sh:43]: We'll add more details in the README in later PR.

**Pull request merged**